src/main/scala/exu/core.scala

```diff
@@ -611,6 +614,13 @@ class BoomCore(implicit p: Parameters) extends BoomModule
   //-------------------------------------------------------------
   // Dispatch to issue queues
 
+  // ist lookup
+  for (w <- 0 until coreWidth) {
+    ist.map(_.io.check(w).addr.valid := dis_fire(w))
+    ist.map(_.io.check(w).addr.bits := dis_uops(w).debug_pc)
+    dis_uops(w).is_lsc_b := ist.map(_.io.check(w).in_ist).getOrElse(false.B)
+  }
+
```

Comment:
I am struggeling a little to track this down, but:
Are you sure you should use dis_fire(w) and not dis_ready as the valid signal? That implies that the REN will dispatch on available ports even when some inst has a hazard. I believe that it uses dis_ready to block all dispatches if there is a hazard on any of them? Or I might be wrong